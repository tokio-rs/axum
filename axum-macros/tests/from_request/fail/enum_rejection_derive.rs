use axum_macros::FromRequest;

#[derive(FromRequest, Clone)]
#[from_request(rejection_derive(!Error))]
enum Extractor {}

fn main() {}
