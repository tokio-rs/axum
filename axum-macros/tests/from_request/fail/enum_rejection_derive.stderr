error: cannot use `rejection_derive` on enums
 --> tests/from_request/fail/enum_rejection_derive.rs:4:16
  |
4 | #[from_request(rejection_derive(!Error))]
  |                ^^^^^^^^^^^^^^^^
